docs/tutorials/configuration.md

```diff
@@ -1,222 +1,7 @@
-
-1. Check which node latency should be configured
-
-   `path.verify()` method, as shown in the following example, tells you which node latency should be defined.
-
-   ```python
-   from caret_analyze import Architecture
-
-   arch = Architecture('yaml', './architecture.yaml')
-   path = arch.get_path('target_path')
-   path.verify()
-
-   ---Output text as below---
-   WARNING : 2021-12-20 19:14:03 | Detected "message_contest is None". Correct these node_path definitions.
-   To see node definition and procedure,execute :
-   >> check_procedure('yaml', '/path/to/yaml', arch, '/message_driven_node')
-   message_context: null
-   node: /message_driven_node
-   publish_topic_name: /topic3
-   subscribe_topic_name: /topic2
-
-   WARNING : 2021-12-20 19:14:03 | Detected "message_contest is None". Correct these node_path definitions.
-   To see node definition and procedure,execute :
-   >> check_procedure('yaml', '/path/to/yaml', arch, '/timer_driven_node')
-   message_context: null
-   node: /timer_driven_node
-   publish_topic_name: /topic4
-   subscribe_topic_name: /topic3
-   ```
-
-   In the example, `path.verify()` tells you two nodes have undefined relationships of input and output.
-
-   - input `/topic2` and output `/topic3` in node `/message_driven_node`
-   - input `/topic3` and output `/topic4` in node `/timer_driven_node`
-
-   Their relationships must be explicit with corresponding message_context items in the architecture file.
-
-2. Define relationship between input and output
-
-   You have to change message_contexts items as below for the sample.
-
-   ```yaml
-   # in /message_driven_node
-   message_contexts:
-     - context_type: use_latest_message # changed from 'UNDEFINED' to 'use_latest_message'
-       subscription_topic_name: /topic2
-       publisher_topic_name: /topic3
-   ```
-
-   ```yaml
-   # in /timer_driven_node
-     message_contexts:　
-     - context_type: use_latest_message # changed from 'UNDEFINED' to 'use_latest_message'
-       subscription_topic_name: /topic3
-       publisher_topic_name: /topic4
-
-   ```
-
-3. Check if node latency is defined
-
-   `path.verify()` tells you that there is no undefined node latency in the path.
-
-   ```python
-   from caret_analyze import Architecture
-
-   arch = Architecture('yaml', './architecture.yaml')
-   path = arch.get_path('target_path')
-   path.verify()
-   ```
-
-   If `path.verify()` returns `True`, CARET can calculate latency of the path. Otherwise, there is any lack of definition to calculate latency.
-
-<prettier-ignore-start>
-!!! todo
-        We'll provide the sample architecture file here, but it's not ready. Sorry for inconvenience.
-<prettier-ignore-end>
+Please refer to [the chapter of Configuration](./configuration.md) if you are interested.
```

Comment:
It looks for me the link is wrong.
```suggestion
Please refer to [the chapter of Configuration](../configuration/practical_example.md) if you are interested.
```

Reply:
@hsgwa  
I'd like to replace `./configuration` by `../configuration.md`, which means the initial page of configuration chapter.

Reply:
fixed at https://github.com/tier4/CARET_doc/pull/112/commits/cacdadc443aefa0396d51be34cef3b1551a80e41